site/content/samples/playwright/coupon.js

```diff
@@ -24,9 +20,9 @@ const productsNumber = process.env.PRODUCTS_NUMBER || 3
   await page.type('#coupon', 'COUPON2020')
   await page.click('.cart > div > button')
 
-  const expectedDiscountedPrice = price * 0.8
-  const discountedPrice = await page.$eval('#total-price', (e) => e.innerText)
-  assert.equal(discountedPrice, expectedDiscountedPrice)
+  const expectedDiscountedPrice = (await price) * 0.8
```

Comment:
@modern-sapien I haven't tried this so I could be wrong: is the `await` needed here?

Reply:
It's not throwing any warnings as far as it not having an effect and I was trying to avoid any potential race conditions.